api/client/formatter/formatter.go

```diff
@@ -140,7 +140,7 @@ ports: {{.Ports}}
 
 	ctx.buffer = bytes.NewBufferString("")
 	ctx.preformat()
-	if ctx.table && ctx.Size {
+	if ctx.table && ctx.Size && !strings.Contains(ctx.Format, "{{.Size}}") {
```

Comment:
You can add a new field on ctx like `sizeSetByFormat` or something and set it in the preprocessor.